docs/articles/expensify-classic/bank-accounts-and-credit-cards/company-cards/Connect-Company-Cards.md

```diff
@@ -1,5 +1,91 @@
 ---
-title: Connect Company Cards
-description: Connect Company Cards
+title: Company-Card-Settings.md
```

Comment:
I think you want to leave this as it was, I believe this gets used here: 
![image](https://github.com/Expensify/App/assets/4073354/3a2f2190-1833-4603-a99f-b73676cf6b7d)


```suggestion
title: Connect Company Cards
```

Reply:
oh maybe i'm wrong here based on Rushat's review.